pysatNASA/instruments/ace_swepam_l2.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""Module for the Advanced Composition Explorer (ACE) EPAM instrument.
+
+References
+----------
+- Stone, E., Frandsen, A., Mewaldt, R. et al. The Advanced Composition Explorer.
+  Space Science Reviews 86, 1–22 (1998). https://doi.org/10.1023/A:1005082526237
+- McComas, D., Bame, S., Barker, P. et al. Solar Wind Electron Proton Alpha
+  Monitor (SWEPAM) for the Advanced Composition Explorer. Space Science Reviews
+  86, 563–612 (1998). https://doi.org/10.1023/A:1005040232597
```

Comment:
Shouldn't references be after Properties?

Reply:
This is consistent with the other instruments in this package with references.  Largely because the description is copy and paste from CDAWeb.  We can add an issue for consistency with other packages.

Reply:
An issue would be good, as this isn't consistent with our stated standards.